src/main/java/uk/gov/beis/els/categories/refrigeratorsdirectsales/model/VendingMachinesForm.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.beis.els.categories.refrigeratorsdirectsales.model;
+
+import org.hibernate.validator.group.GroupSequenceProvider;
+import uk.gov.beis.els.categories.common.StandardTemplateForm20Char;
+import uk.gov.beis.els.categories.internetlabelling.model.InternetLabellingGroup;
+import uk.gov.beis.els.model.meta.DualModeField;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.groups.Default;
+
+@GroupSequenceProvider(VendingMachinesFormSequenceProvider.class)
+public class VendingMachinesForm extends StandardTemplateForm20Char {
+  @FieldPrompt("Energy efficiency class indicator")
+  @NotBlank(message = "Select an energy efficiency indicator", groups = {Default.class, InternetLabellingGroup.class})
+  @DualModeField
+  private String efficiencyRating;
+
+  @FieldPrompt("The annual electricity consumption in kWh in terms of final energy consumption per year")
+  @Digits(integer = 4, fraction = 0, message = "Enter the annual energy consumption, up to 4 digits long")
+  private String annualEnergyConsumption;
+
+  @FieldPrompt("The sum of the net volumes, expressed in litres, of all compartments functioning at chilled operating temperature")
+  @Digits(integer = 4, fraction = 0, message = "Enter the total volume of chilled compartments, up to 4 digits long")
+  private String fridgeCapacity;
+
+  @FieldPrompt("The maximum measured product temperature, expressed in degrees Celcius, of the compartment(s) with chilled operating temperatures")

```

Comment:
Is `Celcius` a british-english thing? I've not seen that before, and it's listed as `Celsius` in the Oxford dictionary, but my spelling is atrocious and I never know the different variants.

Reply:
No my bad, it's a typo. Anders Celsius would be ashamed of me